api/templates/design.go

```diff
@@ -900,13 +900,40 @@ var Instance = Type("instance", func() {
 	Attribute("updated", ChangeInfo, func() {
 		Description("Instance update date and token.")
 	})
+	Attribute("mainConfig", MainConfig)
 	Required("templateId", "instanceId", "branch", "repositoryName", "templateId", "version", "name", "created", "updated")
 })
 
 var InstanceDetail = Type("instanceDetail", func() {
 	Extend(Instance)
 	Attribute("versionDetail", VersionDetail, "Information about the template version. Can be null if the repository or template no longer exists. If the exact version is not found, the nearest one is used.")
-	Required("versionDetail")
+	Attribute("configurations", ArrayOf(Config), "All configurations from the instance.")
+	Required("versionDetail", "configurations")
+})
+
+var MainConfig = Type("mainConfig", func() {
+	Description("Main config of the instance, usually an orchestration. Optional.")
+	Attribute("componentId", String, "Component ID.", func() {
+		Example("keboola.orchestrator")
+	})
+	Attribute("configId", String, "Configuration ID.", func() {
+		Example("7954825835")
+	})
+	Required("componentId", "configId")
+})
```

Comment:
Do definicie instancie som pridal `MainConfig`, je to orchestracia, ktora sa ma spustit, ked sa vytvori sablona, bude tam tlacidlo `run`.